services/src/main/java/org/keycloak/services/resources/admin/AuthenticationManagementResource.java

```diff
@@ -392,6 +392,26 @@ public static void copy(RealmModel realm, String newName, AuthenticationFlowMode
                 execution.setFlowId(copy.getId());
                 copy(realm, newName, subFlow, copy);
             }
+
+            if (execution.getAuthenticatorConfig() != null) {
+                AuthenticatorConfigModel config = realm.getAuthenticatorConfigById(execution.getAuthenticatorConfig());
+
+                if (config == null) {
+                    logger.debugf("Authentication execution with id [%s] not found", config.getId());
+                    throw new IllegalStateException("Authentication execution configuration not found");
+                }
+
+                config.setId(null);
```

Comment:
It is not very nice to duplicate objects like that but changing it would introduce more changes to this PR. Looks like this pattern is being used alot.